app/serializers/review_serializer.rb

```diff
@@ -0,0 +1,5 @@
+class ReviewSerializer < ActiveModel::Serializer
+  attributes :id, :rating, :body, :shop_id, :user_id
```

Comment:
we shouldn't need to include the `user_id` because the belongs to below will handle that for us - not sure if the `shop_id` is needed or used given a review will have an association with the shop